src/Caching/Memory/test/CapacityTests.cs

```diff
@@ -137,7 +137,7 @@ public async Task DoNotAddIfSizeOverflows()
             Assert.Null(cache.Get("key1"));
 
             // Wait for compaction to complete
-            Assert.True(await sem.WaitAsync(TimeSpan.FromSeconds(10)));
+            Assert.True(await sem.WaitAsync(TimeSpan.FromSeconds(20)));
```

Comment:
@BrennanConroy is this a reasonable fix for flakiness in this test class❔ If not you, who should agree or disagree❔

Reply:
Seems fine, we've moved to 30 seconds in a lot of places in 3.1 and 5.0.

Personally, I would change the SemaphoreSlime to a CancellationTokenSource.

Reply:
Let's get this in to unblock any 2.1.28 fixes. Feel free to improve the tests further in another PR.